build.sbt

```diff
@@ -45,7 +46,9 @@ val baseSettings = Seq(
     "org.typelevel" %% "cats-core" % catsVersion,
     "com.twitter" %% "finagle-http" % twitterVersion,
     scalaOrganization.value % "scala-reflect" % scalaVersion.value,
-    "io.catbird" %% "catbird-util" % twitterVersion
+    "io.catbird" %% "catbird-util" % twitterVersion,
```

Comment:
Do we still need catbird dependencies from finch-core?

Reply:
No, actually we don't